README.md

```diff
@@ -125,9 +125,31 @@ Other values are managed by the Vue.js application and are provided in the [`_de
 
 ## Releasing
 
-To publish a new version on the marketplace:
-* Update the version in the main class of this module, and in `config.xml` ifit exists,
-* Create a new release on the [releases page](https://github.com/PrestaShopCorp/pasmarketingwithgoogle/releases) with a tag that matches the version provided in the main class.
+### QA Review
+
+Each version need to be tested by the QA team before being sent to production.
+This step is done once all pull-requests are merged in the base branch (`master` for instance).
+
+To create a pre-release:
+* Check the version in the main class of the module and `config.xml` match the upcoming version, or update the values accordingly,
+* Start creating a new release on the [releases page](https://github.com/PrestaShopCorp/pasmarketingwithgoogle/releases) with a tag that matches the version provided in the main class in the format `v1.XX.XX-beta.XX`.
+* **Make sure to check the option "Set as a pre-release"**
+* A proposal of change-log can be generated by GitHub to list all the merged pull-requests since the previous release. This list can be relevant for the QA team and can be sent as-is.
+* Publish the release.
+
+When created, several zip files will be attached to the release. Each of them is linked to a specific API environment (production, pre-production or integration), and the production zip is the one being sent to the marketplace.
+
+### Deploying to production
+
+Publishing on the marketplace can be done by creating a release on GitHub.
+
+To publish a release:
+* Check the version in the main class of the module and `config.xml` match the upcoming version, or update the values accordingly,
```

Comment:
Pareil pour `psxmarketingwithgoogle.php`, on doit aussi changer la version dans ce fichier il me semble 🤔

Reply:
D'ailleurs j'imagine que c'est vérifier à la CI, non ?

Reply:
The file you mention is "the main class of the module" in the text. I wrote it this way so it can be copy pasted to another module Readme without having to rename files

Reply:
Good to know! I'm writing it down :)